easybuild/main.py

```diff
@@ -425,7 +425,18 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
             sys.exit(31)  # exit -> 3x1t -> 31
 
     # read easyconfig files
-    easyconfigs, generated_ecs = parse_easyconfigs(paths, validate=not options.inject_checksums)
+    try:
+        easyconfigs, generated_ecs = parse_easyconfigs(paths, validate=not options.inject_checksums)
+    except Exception as e:
```

Comment:
@branfosj Please use `err` rather than `e` (single-letter variables are considered evil)

Reply:
done in c93efc2